### PR TITLE
virtio_transitional: Add test case for virtio-serial

### PR DESCRIPTION
**Pull request by vivianQizhu**
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
(1/3) type_specific.io-github-autotest-libvirt.virtio_transitional_serial.virtio.with_pcie_to_pci_bridge: PASS (52.65 s)
 (2/3) type_specific.io-github-autotest-libvirt.virtio_transitional_serial.virtio_transitional.default: PASS (47.04 s)
 (3/3) type_specific.io-github-autotest-libvirt.virtio_transitional_serial.virtio_transitional.with_pcie_to_pci_bridge: PASS (51.38 s)

File: libvirt/tests/src/virtio_transitional/virtio_transitional_serial.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        cat_cmd = "cat %s" % host_path
+        logfile = "test_data_transfer-%s.log" % dev_type
+        host_process = aexpect.ShellSession(cat_cmd, auto_close=False,
+                                            output_func=utils_misc.log_line,
+                                            output_params=(logfile,))
+        guest_session = vm.wait_for_login()
+        guest_session.cmd_output('echo %s > %s' % (test_message, guest_path))
+        guest_session.close()
+        try:
+            host_process.read_until_last_line_matches(test_message, timeout=10)
```
Comment (kylazhang) [inline libvirt/tests/src/virtio_transitional/virtio_transitional_serial.py]: I think here should add exception to get the exception and do test.fail with clear failed reason

**Comment by vivianQizhu** [comment]
@kylazhang Updated, thanks.

**Comment by kylazhang** [comment]
Thanks for all the comments, as vivianQizhu will be offline for more than 4 month, I suggest to merge the new file first and then submit the fix by others.

**Pull request merged**